app/models/user.rb

```diff
@@ -2,4 +2,7 @@ class User < ApplicationRecord
   validates_presence_of :name, :email, :password
   has_many :reviews
   has_many :notes
+  has_many :created_sodas, :class_name => 'Soda', :foreign_key => 'creator_id'
+  has_many :last_edited_sodas, :class_name => 'Soda', :foreign_key => 'last_editor_id'
```

Comment:
Can you provide a use case where we would want this information from the user's side? I thought about it, and couldn't come up with one, so I purposefully omitted this side of the relationship. Should a user care about the last new soda they added into the db? Or the last one they edited? We definitely need it from the soda side, but a user doesn't care